config/environments/test.rb

```diff
@@ -28,9 +28,6 @@
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
-  # Raise exceptions instead of rendering exception templates
-  config.action_dispatch.show_exceptions = false
-
```

Comment:
I see this is needed to check for the 404 in the test. So in this case there is probably an exception in the log/test.log file.  Can you add a comment to the test(s) that require this change, incase someone tries to add `show_exceptions = true` back in?
This way when the tests start failing, the developer might figure out why more quickly.